services/core-api/app/config.py

```diff
@@ -52,6 +53,10 @@ class Config(object):
     NRIS_USER_NAME = os.environ.get('NRIS_USER_NAME', None)
     NRIS_PASS = os.environ.get('NRIS_PASS', None)
     ENVIRONMENT_NAME = os.environ.get('ENVIRONMENT_NAME', 'dev')
+    CORE_PRODUCTION_URL = os.environ.get('CORE_PRODUCTION_URL',
```

Comment:
I did not set up these 3 env vars. I figure they are the same in each namespace (dev/test/prod). Do we want to do that @jon-funk?

Reply:
Should be fine as is.